### PR TITLE
Annotation prepare home page

### PR DESCRIPTION
**Pull request by antoinemrcr**
there is a button to prepare all annotations, and also, if you click on annotate, it will prepare annotations before.
But you need to launch the script from the project now, and add the pythonpath

### CODE REVIEW

File: sense/finetuning.py [hunk 1]
```diff
@@ -217,6 +218,28 @@ def compute_features(video_path, path_out, inference_engine, num_timesteps=1, pa
...
+    frames_folder = join(dataset_path, f'frames_{split}', label)
+    os.makedirs(features_folder, exist_ok=True)
+    os.makedirs(frames_folder, exist_ok=True)
+
+    # Loop through all videos for the given class-label
+    videos = glob.glob(folder + '/*.mp4')
+    for e, video_path in enumerate(videos):
+        print(f"\r  Class: \"{label}\"  -->  Processing video {e + 1} / {len(videos)}", end="")
+        path_frames = join(frames_folder, video_path.split("/")[-1].replace(".mp4", ""))
+        path_features = join(features_folder, video_path.split("/")[-1].replace(".mp4", ".npy"))
```
Comment (corneliusboehm) [inline sense/finetuning.py:236]: I would suggest `os.path.basename()` instead of `split("/")`

**Review by corneliusboehm — COMMENTED** [review]
Quick review

File: scripts/tools/annotation/prepare_annotation.py [hunk 2]
```diff
@@ -19,8 +19,7 @@
 
 from sense import engine
 from sense import feature_extractors
-from sense.finetuning import compute_features
-from sense.finetuning import set_internal_padding_false
+from sense.finetuning import compute_frames_features
 
 if __name__ == "__main__":
```
[the remark this answers is not in the document]
Comment (antoinemrcr) [reply scripts/tools/annotation/prepare_annotation.py:24]: Maybe not indeed

File: scripts/tools/annotation/templates/up_project_details.html [hunk 3]
```diff
@@ -7,6 +7,14 @@ <h1 class="ui header">{{ config.name }}</h1>
...
     Path: {{ path }}
 </h3>
+<a class="ui icon button" onclick="spinner()" href="{{ url_for('prepare_annotation', path=path) }}">
+    <div class="loader">
+        <div class="loading">
+        </div>
+    </div>
+    <i class="edit icon"></i>
+    prepare annotations
+</a>
```
[the remark this answers is not in the document]
Comment (antoinemrcr) [reply scripts/tools/annotation/templates/up_project_details.html:17]: there is some css and javascript.
the loading is not on the button, but on the middle of the page

[the remark this answers is not in the document]
Comment (corneliusboehm) [reply scripts/tools/annotation/templates/up_project_details.html:17]: I just tested it and the waiting spinner was hidden behind another element, so I think it's safer if we use the one on the button

**Pull request merged**